nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -230,27 +230,55 @@ xkb_symbols &quot;us-greek&quot;
   key &lt;LatZ&gt; { [ z, Z, Greek_zeta  ] };
 };
 </programlisting>
+  <note>
   <para>
-   To install the layout, the filepath, a description and the list of
-   languages must be given:
+   The longest layout name will be the width of the layout indicator
+   widget.
+  </para>
+  </note>
+  <para>
+   A minimal layout specification must include the following:
   </para>
 <programlisting>
 <xref linkend="opt-services.xserver.extraLayouts"/>.us-greek = {
   description = "US layout with alt-gr greek";
   languages   = [ "eng" ];
-  symbolsFile = /path/to/us-greek;
+  symbolsFile = /yourpath/symbols/us-greek;
 }
 </programlisting>
   <note>
   <para>
-   The name should match the one given to the
-   <literal>xkb_symbols</literal> block.
+   The name <literal>us-greek</literal> must match the one given to
+   the <literal>xkb_symbols</literal> block.
+  </para>
+  </note>
+  <note>
+  <para>
+   Make sure that your XKB file compiles and works fine!
```

Comment:
I would put this note *after* the instruction for testing the layout.

Reply:
i'm happy to reorder it, but my motivation to put it early was that some people do the instructions as they are reading it.

does that change your mind?

Reply:
It makes sense, but it makes me think "I have to search how layouts are compiled".  Maybe add a "See below".

Reply:
how about this: i have deleted the note, but made part of the text bold. (PR updated)